docker/requirements.txt

```diff
@@ -18,8 +18,9 @@ kubernetes
 libsass
 pip
 pylint
-pymysql
+PyMySQL==0.9.*
```

Comment:
Modified because the "correct" package name is PyMySQL (https://pymysql.readthedocs.io/en/latest/user/installation.html) and we should start pinning versions.